src/client/Client.cc

```diff
@@ -11690,6 +11705,62 @@ int Client::ll_removexattr(Inode *in, const char *name, const UserPerm& perms)
   return _removexattr(in, name, perms);
 }
 
+int Client::worm_state_transition(Inode *in, const UserPerm& perms, int op)
```

Comment:
A comment header would be nice here. What, specifically, is this function intended to do?

Reply:
OK